### PR TITLE
Windows try

### PR DESCRIPTION
**Pull request by arghyatiger**
An attempt at making Apertium+Python pass the Appveyor Test for Windows build

### CODE REVIEW

**Comment by arghyatiger** [comment]
@sushain97 

```

C:\Users\User\Documents\apertium-python-master\apertium-python-master>echo "^cat<n><pl>$" | lt-proc -g .\apertium-all-dev\share\apertium\apertium-eng\eng.autogen.bin

```

the following command gives no output on windows. returns blank.
the command seems to be correct to me. And it is running, but no output is returned

**Comment by arghyatiger** [comment]
@sushain97 

```echo "^cat<n><pl>$" | lt-proc -g 'path to eng.autogen.bin'```

returns nothing on the windows terminal

**Comment by arghyatiger** [comment]
```

C:\Users\User\Documents\apertium-python-master\apertium-python-master\apertium-all-dev\share\apertium\apertium-eng>lt-comp lr apertium-eng.eng.dix eng.automorf.bin
contractions@standard 434 505
final@inconditional 110 2972
main@standard 68747 130701
```
Tests give results as well

**Comment by sushain97** [comment]
There are a number of unrelated changes in this PR. Please remove them so I can review it effectively.

File: .appveyor.yml [hunk 1]
```diff
@@ -3,22 +3,22 @@ cache:
   - '%LOCALAPPDATA%\pip\Cache'
 environment:
   matrix:
-    - PYTHON: C:\\Python34
+    - PYTHON: c:\python34
```
Comment (sushain97) [inline .appveyor.yml]: The standard in Windows is to capitalize drive letters so I think these should all be changed back to how they were.

**Comment by arghyatiger** [comment]
@sushain97  Before you go forward with reviewing, there's a more fundamental problem I'm facing.

The ```echo "^cat<n><pl>$" | lt-proc -g 'path to eng.autogen.bin'``` runs but doesn't return outptut.

When can this happen?

**Comment by sushain97** [comment]
Is there a verbose mode you can use to see output and does the exact same invocation work on Linux? I'm not really sure why that would happen and even less so on Windows so you'll probably want to ask in #apertium.

**Comment by arghyatiger** [comment]
I did ask on #apertium. I'll try writing a mail to Tino.

I'll look for the verbose mode

**Comment by arghyatiger** [comment]
@sushain97 

Another interesting observation

``` echo "cats" | lt-proc 'path to eng.automorph.bin' ```
returns the correct output

but

```echo "^cat<n><pl>$" | lt-proc -g 'path to eng.autogen.bin' ``` doesn't

**Comment by arghyatiger** [comment]
@sushain97 

Also, I tried running the ```en-es.mode``` commands manually


```

C:\Users\User\Documents\apertium-python-master\apertium-python-master>echo "cats"|  lt-proc .\apertium-all-dev\share\apertium\apertium-en-es\en-es.automorf.bin | apertium-tagger -g $2 .\apertium-all-dev\share\apertium\apertium-en-es\en-es.prob | apertium-pretransfer| apertium-transfer -n .\apertium-all-dev\share\apertium\apertium-en-es\apertium-en-es.en-es.genitive.t1x  .\apertium-all-dev\share\apertium\apertium-en-es\en-es.genitive.bin | lt-proc -b .\apertium-all-dev\share\apertium\apertium-en-es\en-es.autobil.bin | lrx-proc -m .\apertium-all-dev\share\apertium\apertium-en-es\en-es.autolex.bin | apertium-transfer -b .\apertium-all-dev\share\apertium\apertium-en-es\apertium-en-es.en-es.t1x  .\apertium-all-dev\share\apertium\apertium-en-es\en-es.t1x.bin | apertium-interchunk .\apertium-all-dev\share\apertium\apertium-en-es\apertium-en-es.en-es.t2x  .\apertium-all-dev\share\apertium\apertium-en-es\en-es.t2x.bin | apertium-postchunk .\apertium-all-dev\share\apertium\apertium-en-es\apertium-en-es.en-es.t3x  .\apertium-all-dev\share\apertium\apertium-en-es\en-es.t3x.bin | lt-proc $1 .\apertium-all-dev\share\apertium\apertium-en-es\en-es.autogen.bin | lt-proc -p .\apertium-all-dev\share\apertium\apertium-en-es\en-es.autopgen.bin
apertium-tagger: can't open SERIALISED_TAGGER file "$2"
Try 'apertium-tagger --help' for more information.
Error: Cannot not open file '$1'.
```

Getting the following error

**Comment by arghyatiger** [comment]
@sushain 

So I tried running  ```lt-comp``` for  '''eng.automorph.bin```  and '''eng.autogen.bin```.
The commands seem to work fine after doing so.

So, I'll have to be using the Popen command to run the lt-comp on the files too.
And the code has to be generic I guess.
Because different language packages will have different parameters.
So the should I just write another module that does the language bin compilations?

**Comment by sushain97** [comment]
If I recall correctly, you're trying to use the Debian versions of compiled language data with the Windows versions of lttoolbox. I'm not sure that would work as expected...

When you used `lt-comp` to compile it yourself on Windows, that worked as expected.

> So the should I just write another module that does the language bin compilations?

You'd have to either download Windows compatible versions of the compiled language data or compile it all on the Windows machine which isn't acceptable because of the number of extra dependencies it requires.

cc @TinoDidriksen

**Comment by arghyatiger** [comment]
@sushain97 

I did talk to Tino. The .deb files have windows binaries in them. But they should run without need of compilation. 

Something is wrong and I'm trying to find out what

**Comment by arghyatiger** [comment]
@sushain97 

Tino says that the need for ```lt-comp`` is only temporary untill the new weights feature in lt-toolbox is fixed. :)

**Pull request closed without merging**